libs/rbcapi/src/bills/managePayees.test.ts

```diff
@@ -0,0 +1,58 @@
+
+import  { getEnvVars } from "../../../../tools/setenv";
+import { deletePayee, addPayee } from './managePayees';
+import { describe, IsManualRun } from '@thecointech/jestutils';
+import { getPayeeOptions, openBillPaymentPage } from '.';
+import { ApiAction, initBrowser } from '../action';
+
+const vars = getEnvVars("prod");
+
+jest.setTimeout(1000 * 60 * 1000);
+
+// We run this test on the live website to catch any changes to RBC website.
+describe("Live website testing", () => {
+
+  const fakeVisaNumber = "4520356320843708";
+  const fakeVisaIssuer = "VISA - TORONTO DOMINION";
+  const fakeVisaName = "testing fakevisa";
+
+  beforeAll(async () => {
+    process.env.RBCAPI_CREDENTIALS_PATH = vars.RBCAPI_CREDENTIALS_PATH;
+    ApiAction.initCredentials();
+    await initBrowser({headless: !IsManualRun});
+  })
+
+  it ('throws on non-unique name', async () => {
+    const act = await openBillPaymentPage("test should-throw");
+    await expect(addPayee(act, fakeVisaName, "VISA", fakeVisaNumber)).rejects.toThrow("DUPLICATE-NAME-ERROR");
+  })
+
+  it ('throws on invalid acc number name', async () => {
+    const invalidVisaNumber = "4520356320843700";
+    const act = await openBillPaymentPage("test should-throw");
+    await expect(addPayee(act, fakeVisaName, fakeVisaIssuer, invalidVisaNumber)).rejects.toThrow("SET-ACC-NO-FAILED");
+  })
+
+
+  it('can add and remove a payee', async () => {
```

Comment:
it would be nicer to split this into 2 tests, but as I dont want to leave random fake visas on my bank account, they have to be merged into one